rasa/core/training/interactive.py

```diff
@@ -1616,6 +1607,12 @@ def run_interactive_learning(
     # before_server_start handlers make sure the agent is loaded before the
     # interactive learning IO starts
     if server_args.get("model"):
+        unpacked_model = model.get_model(server_args.get("model"))
```

Comment:
you can use `server_args.get("core")` and `server_args.get("nlu")` to avoid unpacking it again

Reply:
amazing